### DISCUSSION TITLE
Is it posssible to generate demo content?

### DISCUSSION THREAD

**Discussion by ugintl** [kimai/kimai#2957] [Questions & Answers] [answered]
I want to generate some demo content. Is it available?

**Comment by kevinpapst** [comment, marked as answer]
```
bin/console doctrine:fixtures:load
```

But you need to run a development setup, see https://www.kimai.org/documentation/developers.html

**Comment by ugintl** [comment]
Once I import the data, should I change it from dev to prod?

**Comment by ugintl** [comment]
@kevinpapst What is the purpose of this command `bin/console kimai:reset:dev`?
